Packs/HackerOne/Integrations/HackerOne/HackerOne.py

```diff
@@ -0,0 +1,588 @@
+        if not key or not value:
+            continue
+
+        if "[]" in key:
+            params[key] = argToList(value)
+        else:
+            params[key] = value
+
+    return params
+
+
+def validate_fetch_incidents_parameters(max_fetch: Optional[int], program_handle: List, filters: str):
+    """
+    Validates fetch incident parameters, raise ValueError on invalid arguments.
+
+    :type max_fetch: ``int``
+    :param max_fetch: Maximum number of incidents per fetch provided by user.
+
+    :type program_handle: ``List``
+    :param program_handle: The program handle provided by the user.
+
+    :type filters: ``str``
+    :param filters: The advanced_filter argument provided by the user.
+
+    """
+    if not 0 < max_fetch <= 100:  # type:ignore
+        raise ValueError(MESSAGES["INVALID_MAX_FETCH"].format(max_fetch))
+
+    if not program_handle:
+        raise ValueError(MESSAGES['PROGRAM_HANDLE'])
+
+    if filters:
+        try:
+            json.loads(filters)
+        except (json.JSONDecodeError, json.decoder.JSONDecodeError, AttributeError):
+            raise ValueError(MESSAGES["FILTER"])
+
+
+def prepare_fetch_incidents_parameters(max_fetch, first_fetch, program_handle, severity, state, filters) -> \
+        Dict[str, Any]:
+    """
+    Prepare fetch incidents params
+    :type max_fetch: ``int``
+    :param max_fetch: Maximum number of incidents per fetch provided by user.
+
+    :type first_fetch: ``str``
+    :param first_fetch: Date or relative timestamp to start fetching incidents from.
+
+    :type program_handle: ``List``
+    :param program_handle: The program handle provided by the user.
+
+    :type severity: ``str``
+    :param severity: Severity level provided by user.
+
+    :type state: ``str``
+    :param state: State provided by user.
+
+    :type filters: ``str``
+    :param filters: The advanced_filter argument provided by the user.
+
+    """
+
+    fetch_params: Dict[str, Any] = {"page[size]": max_fetch, "sort": "reports.created_at"}
+
+    fetch_params.update(
+        prepare_filter_by_arguments(program_handle, severity, state, filters))
+
+    fetch_params["filter[created_at__gt]"] = arg_to_datetime(first_fetch).strftime(DATE_FORMAT)  # type:ignore
+
+    return assign_params(**fetch_params)
+
+
+def validate_common_args(args: Dict[str, str]):
+    """
+    Validate page_size and page_number argument, raise ValueError on invalid arguments.
+
+    :type args: ``Dict[str, str]``
+    :param args: The command arguments provided by the user.
+    """
+
+    page_size = arg_to_number(args.get("page_size", 50))
+    if page_size is not None:
+        if page_size <= 0 or page_size > 100:
+            raise ValueError(MESSAGES['PAGE_SIZE'].format(page_size))
+
+    page_number = arg_to_number(args.get("page_number"))
+    if page_number is not None:
+        if page_number < 0 or page_number > INT32:
+            raise ValueError(MESSAGES['PAGE_NUMBER'].format(page_number))
```

Comment:
See my comment in line 462
```suggestion
def validate_common_args(**args: Dict[str, str]):
    """
    Validate page_size and page_number argument, raise ValueError on invalid arguments.

    :type args: ``Dict[str, str]``
    :param args: The command arguments provided by the user.
    """

    page_size =args.get("page_size")
    if page_size is not None:
        if page_size <= 0 or page_size > 100:
            raise ValueError(MESSAGES['PAGE_SIZE'].format(page_size))

    page_number =args.get("page_number")
    if page_number is not None:
        if page_number < 0 or page_number > INT32:
            raise ValueError(MESSAGES['PAGE_NUMBER'].format(page_number))
```